src/app/modules/frontend/frontend.server.controller.ts

```diff
@@ -97,3 +114,16 @@ module.exports.redirectLayer = function (req, res) {
     })
   }
 }
+
+/**
+ * Handler for GET /frontend/features endpoint.
+ * @param req - Express request object
+ * @param res - Express response object
+ * @returns {String} Current featureManager states
+ */
+export const showFeaturesStates: RequestHandler<
+  ParamsDictionary,
```

Comment:
sorry one more thing, I realised that all these controllers still use the old method of supplying `ParamsDictionary` as the first generic type argument, which we don't want as per #948. could you help change all of these to `unknown` pls?

Reply:
Ok edited; apart from `showFeaturesStates`, the rest require `ParamsDictionary` as the first generic type argument because they call `createReqMeta(req)` and req is of interface `Request` which requires `ParamsDictionary` as the first generic type argument